cors-policy/pom.xml

```diff
@@ -24,9 +24,28 @@
       <artifactId>apiman-gateway-engine-policies</artifactId>
       <scope>provided</scope>
     </dependency>
+    
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
```

Comment:
Yep, that seems the right way to do it to me as well. I've made a PR, and will update the CORS testing PR when that's merged.